### ISSUE TITLE
[3.3.5] DB / locales_quest "Nessa Shadowsong"

### ISSUE THREAD

**Issue by lyn1337** [TrinityCore/TrinityCore#17345]
core_version: TrinityCore rev. 7db377c7f974 2016-06-09 17:00:00 +0100 (3.3.5 branch) (Unix, Release)
db_ version: TDB 335.61

```sql
-- [3.3.5]Quest "Nessa Shadowsong" (entry: 6344)
-- source: http://wowwiki.wikia.com/wiki/Quest:Nessa_Shadowsong
-- remove all wrong locales for this quest
UPDATE `locales_quest` SET `Title_loc2`='', `Title_loc3`='', `Title_loc7`='', `Title_loc8`='', `Details_loc2`='', `Details_loc3`='', `Details_loc7`='', `Details_loc8`='', `Objectives_loc2`='', `Objectives_loc3`='', `Objectives_loc7`='', `Objectives_loc8`='' WHERE `Id`=6344;
```

edit: only loc6 is correct.

**Comment by MrSmite** [comment]
I don't think this is a 3.3.5 quest, there is no Hippogryph Master in Dolanaar.

**Comment by SnapperRy** [comment]
This quest's title is actually "Nessa Shadowsong" in 3.3.5. Name was changed in 4.0.3.

More info: http://wowwiki.wikia.com/wiki/Quest:Nessa_Shadowsong

**Comment by MrSmite** [comment]
> Existing data is right, sorry @lyn1337 :P

No, it's still not correct. Nessa Shadowsong starts with NPC 4241 http://www.wowhead.com/?npc=4241, the SQL above has an NPC listed that does not exist in 3.3.5

**Comment by SnapperRy** [comment]
Yeah, I meant that current TDB data is right, not this fix :P

**Comment by lyn1337** [comment]
@SnapperRy @MrSmite

**Comment by lyn1337** [comment]
@SnapperRy If it really got renamed, the TDB data is more than messed up.

**Comment by lyn1337** [comment]
@MrSmite Youve got the wrong local text.

**Comment by tkrokli** [comment]
Thanks for this reminder, that some of the wowhead quest pages keep old and wrong values while the page content is updated to match the new content. it is a two-edged sword, I guess, that Blizzard changes quest content, but keeps the old quest IDs :-P

**Comment by SnapperRy** [comment]
In 3.3.5, the quest is named "Nessa Shadowsong". In Cataclysm (4.0.3) the quest id was reused and the quest renamed to "Reminders of Home".

But for 3.3.5, Nessa Shadowsong has quest giver 4241 and is already right on TDB.

**Comment by lyn1337** [comment]
Currently the quest is named after the cata one in TDB. This has confused me a bit, I wasn't too sure about the giver. 
Then it's just the locales being totally wrong.

**Comment by lyn1337** [comment]
@SnapperRy Thanks for clarifying the correct giver, I really should get an english client.

**Comment by Aokromes** [comment]
Why no simply fix names instead removing?

**Comment by lyn1337** [comment]
@Aokromes soon™

**Comment by lyn1337** [comment]
@Aokromes Source?

**Comment by lyn1337** [comment]
@Aokromes Should be ok for title_loc, I guess.

**Comment by MrSmite** [comment]
> Youve got the wrong local text.

I think you meant this for someone else? I use the default locale and the Nessa Shadowsong quest has worked since forever on TC.

**Comment by lyn1337** [comment]
@MrSmite 

2. Fidelio is mentioned in the incorrect localization. 
3. The quest shouldn't be removed.
4. 4265 does exist

**Comment by MrSmite** [comment]
@lyn1337 I'm not sure what your point is.

1. Your original SQL was changing NPC 4241 to NPC 4265 for quest 6344 which is incorrect.
2. Your original post linked to the cataclysm version of quest 6344 but you're referring to the 3.3.5 DB. I said it should be removed because the cataclysm version of the quest doesn't belong in 3.3.5
3. I corrected you by saying that 4265 does not exist **in Dolanaar** in 3.3.5

You originally mixed up the 3.3.5 and 4.x version of the quest and I was trying to set you straight by showing you the valid data for 3.3.5 which already exists in the DB.

**Comment by lyn1337** [comment]
@MrSmite 
What's the point in mentioning Fidelio then? Or what's your point now?

1. It was based on the many localized texts and changed immediately.
2. Then I got that point of yours wrong, you weren't really clear about anything.
3. Quoting you: "an NPC that does not exist in 3.3.5"

TDB mixes up both wotlk and cata versions of the quest and this is why I got confused by it. Because I thought these texts were checked of some sort when they got added. Just because the english data is correct, it doesn't mean the rest is as well. So don't call it correct, without even having a quick look at it. Not all are using just the english version.
